### PR TITLE
Fixed duplicating entries bug

### LINKED ISSUE #49: [Export] Duplicate entries getting exported while performing export
The current implementation duplicates the `MUST_INCLUDED_ATTRIBUTES` while exporting the content from the site.

### PR DESCRIPTION
**Pull request by ironmaniiith**
Fixes #49 

Note: If it's not a mandatory requirement to have `MUST_INCLUDED_ATTRIBUTES` in the beginning, then the code can be improved further.
@Shriyanshagro @djay please review

### CODE REVIEW

**Comment by Shriyanshagro** [comment]
@ironmaniiith it is not mandatory to have it at the beginning. But it is mandatory to have this list as far as I remember. Make sure, you don't break the code.

**Comment by djay** [comment]
What is manditory has certainly changed.

Your original code required a lot for no reason. All that was taken out. Pretty much nothing is mandatory now

**Comment by ironmaniiith** [comment]
I checked for any test which might be failing after resolving this bug. There were none. So I went ahead and fixed it.

**Comment by djay** [comment]
I don't see an explanation of what the bug is

**Comment by ironmaniiith** [comment]
> I don't see an explanation of what the bug is

Okay, I'll add details

File: src/plone/importexport/browser/importexport.py [hunk 1]
```diff
@@ -189,16 +189,15 @@ def export(self):
...
 
-            # MUST_INCLUDED_ATTRIBUTES must present in headers and that too
-            # at first position of list
-            for element in reversed(MUST_INCLUDED_ATTRIBUTES):
-                headers.insert(0, element)
+            # Just to make sure headers does not contain any attribute
+            # from MUST_INCLUDED_ATTRIBUTES
+            headers = list(set(headers) - set(MUST_INCLUDED_ATTRIBUTES))
+            # Appending MUST_INCLUDED_ATTRIBUTES in the beginning
+            headers = MUST_INCLUDED_ATTRIBUTES + headers
```
Comment (ironmaniiith) [inline src/plone/importexport/browser/importexport.py]: @djay , as you said, if it's not mandatory to have `MUST_INCLUDED_ATTRIBUTES` in the headers, I can remove this line in a different commit?

**Comment by djay** [comment]
Yep. Feel free to clean up what doesn't make sense as you go.  It should
have been done before in the commit that got rid of the compulsory fields.

**Comment by ironmaniiith** [comment]
Description: While exporting the contents from the site, there are some fields which were originally *mandatory* fields and were getting repeated in the metadata. Hence the metadata looks something like: 

Screenshot from 2019-06-20 19-54-59

As we can see, fields such as `@type`, `path`, `id`, `UID` were getting repeated which is not a desired behaviour

This PR aims to resolve this behaviour

**Pull request merged**